.github/workflows/benchmark.yml

```diff
@@ -13,13 +13,13 @@ jobs:
     outputs:
       PR-BENCH-14: ${{ steps.benchmark-pr.outputs.BENCH_RESULT14 }}
       PR-BENCH-16: ${{ steps.benchmark-pr.outputs.BENCH_RESULT16 }}
-      PR-BENCH-17: ${{ steps.benchmark-pr.outputs.BENCH_RESULT17 }}
+      PR-BENCH-17: ${{ steps.benchmark-pr.outputs.BENCH_RESULT18 }}
```

Comment:
```suggestion
      PR-BENCH-18: ${{ steps.benchmark-pr.outputs.BENCH_RESULT18 }}
```

Reply:
Node 17 is still supported until 2022-06-01, may as well hang onto it?

Reply:
I think we just need to support LTS versions.